modules/exploits/linux/http/webmin_backdoor.rb

```diff
@@ -94,6 +94,11 @@ def check
       return CheckCode::Unknown
     end
 
+    if res.body.include?('This web server is running in SSL mode.')
+      print_error('Please enable the SSL option to proceed')
```

Comment:
`print_error` for visibility.

Reply:
`datastore['SSL'] = true` would be nice, but it's too chaotic. I think a non-verbose print is good.